test/utils.js

```diff
@@ -165,6 +165,22 @@ const fixtures = {
 		return Promise.all(Array.from({
 			 length: numberOfTags 
 		}).map(() =>  fixtures.insertTag()))
+	},
+	async insertPostComment(overrides = {}){
+		const commentData = {
+			comment : faker.random.words()
+		}
+		const newData = {...commentData, ...overrides}
+
+		const queryResult = await db.query(
+			`INSERT INTO comments 
+			("userId" , "postId" , content)
+			 VALUES ($1 , $2 ,$3) RETURNING *`,
+			[newData.userId, newData.id, newData.comment]
+		)
+
+		const insertedComment = queryResult.rows[0]
+		return {insertedComment}
```

Comment:
Its the only data we are returning so let's it on its own instead of in an object

```suggestion
	 return queryResult.rows[0]
```